### PR TITLE
test for links

### PR DESCRIPTION
**Pull request by kellyselden**
A test for links mentioned introduced in #1096 and mentioned in #1108. I branched off the commit before the breaking commit, so CI should pass, but if you merge, it will fail.

cc @lancedikson

### CODE REVIEW

File: tests/dummy/mirage/serializers/blog-post.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import { JSONAPISerializer } from 'ember-cli-mirage';
 
 export default JSONAPISerializer.extend({
+  links(blogPost) {
+    return {
+      comments: {
+        related: `/blog-posts/${blogPost.id}/comments`
```
Comment (lancedikson) [inline tests/dummy/mirage/serializers/blog-post.js]: @kellyselden, could you please update the path here as well to `/api/blog-posts/${blogPost.id}/comments`?

Comment (lancedikson) [inline tests/dummy/mirage/serializers/blog-post.js]: Thanks :)

**Comment by samselikoff** [comment]
Sorry for the late reply...

I just checked this out locally and the test didn't break. @kellyselden @lancedikson Do we know if this is still an issue?

**Comment by samselikoff** [comment]
I believe this somehow sorted itself out, so closing for now. Happy to reopen if still an issue.

**Pull request closed without merging**